tokio/src/runtime/tests/loom_local.rs

```diff
@@ -0,0 +1,39 @@
+use crate::runtime::tests::loom_oneshot as oneshot;
+use crate::runtime::Builder;
+use crate::task::LocalSet;
+
+use std::task::Poll;
+
+#[test]
+fn wake_during_shutdown() {
```

Comment:
it would be nice to have a comment explaining what the failure modes of this test are, for future reference? not a blocker though.